be/src/storage/lake/update_manager.cpp

```diff
@@ -362,6 +369,36 @@ Status UpdateManager::get_column_values(Tablet* tablet, const TabletMetadata& me
             RETURN_IF_ERROR(col_iter->fetch_values_by_rowid(rowids.data(), rowids.size(), (*columns)[i].get()));
         }
     }
+    if (state != nullptr && with_default) {
```

Comment:
This part is just same as above? Only tablet schema is different? Can you wrap this logic to a function and reduce duplicate code?

Reply:
done, use lamda instead